src/components/Main.test.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import TestRenderer from 'react-test-renderer';
+import { MemoryRouter } from 'react-router-dom'
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider'
+import { AppBar } from 'material-ui';
+import Main from './Main';
+
+test('can render /assets/dept', () => {
+  const testInstance = TestRenderer.create(
+    // TODO you should be able to use initialEntries/initialIndex - doesn't seem to add match property
```

Comment:
This call to ``TestRenderer.create()`` would appear to be copy-and-pasted in many placed with only the URL changing. Can we refactor it into a factory function?